framework/src/modules/interoperability/base_interoperability_store.ts

```diff
@@ -45,6 +57,36 @@ export abstract class BaseInteroperabilityStore {
 		console.log(!this._moduleID, !this._interoperableModules, !this.getStore);
 	}
 
+	public async appendToOutboxTree(chainID: Buffer, appendData: Buffer) {
+		const channelSubstore = this.getStore(MODULE_ID_INTEROPERABILITY, STORE_PREFIX_CHANNEL_DATA);
+		const channel = await channelSubstore.getWithSchema<ChannelData>(chainID, channelSchema);
+		const updatedOutbox = regularMerkleTree.calculateMerkleRoot({
+			value: hash(appendData),
+			appendPath: channel.outbox.appendPath,
+			size: channel.outbox.size,
+		});
+		await channelSubstore.setWithSchema(
+			chainID,
+			{ ...channel, outbox: updatedOutbox },
+			channelSchema,
+		);
+
+		return true;
```

Comment:
why is this returning `true`? same as below

Reply:
The LIP states to return true.